alias.yml

```diff
@@ -3,21 +3,21 @@
 # Usage: gh <alias> <args>
 
 # Workflow
+# These commands basically save you from typing the full "dxp" command each time.
 branch: dxp branch
 clone: repo clone
 lint: dxp lint
-merge: dxp merge
+owner: dxp owner
+pr-create: dxp pr-create
+pr-list: pr list
+pr-merge: dxp pr-merge
+pr-view: pr view
+status: dxp status
+template: dxp template
 test: dxp test
 
-# Pull Requests
-prc: dxp pr
-prm: dxp merge
-prs: pr list
+# More concise PR aliases
+prc: dxp pr-create
```

Comment:
Might be personal preference, but they way I read this is that we have modifiers of the PR-command here. Meaning i read this as 

<command> <target> <modifier> etc.

In this case i think the list, create, merge etc should be part of the pr subcommand and should therefore be

- dxp pr (doesn't do anything as it has no modifier - maybe help text, w/e)
- dxp pr create
- dxp pr list
- dxp pr merge
- dxp pr view

Another alternative is to do something akin to the kubectl command where you have <command> <modifier> <target> (e.g. kubectl get pods) (i think this works better if you have many targets)

This also makes the aliases flow a bit more smoothly imo - like pr is short for dxp pr, prl is short for pr list etc (making nesting a bit easier)

Reply:
Hmm... I can buy that. Future PR.